hat/menupermissions/models.py

```diff
@@ -66,57 +95,57 @@ class CustomPermissionSupport(models.Model):
 
     # Used in setup_account api
     DEFAULT_PERMISSIONS_FOR_NEW_ACCOUNT_USER = [
-        "iaso_forms",
-        "iaso_submissions",
-        "iaso_mappings",
-        "iaso_completeness",
-        "iaso_org_units",
-        "iaso_links",
-        "iaso_users",
-        "iaso_projects",
-        "iaso_sources",
-        "iaso_data_tasks",
-        "iaso_reports",
+        _FORMS,
+        _SUBMISSIONS,
+        _MAPPINGS,
+        _COMPLETENESS,
+        _ORG_UNITS,
+        _LINKS,
+        _USERS,
```

Comment:
Here, you could fix another existing ticket by including all the permissions for a new account_user

Reply:
Could you link the ticket here?